life_expectancy/cleaning.py

```diff
@@ -26,6 +20,17 @@ def clean_data(country):
     dataframe = dataframe.melt(id_vars=['unit','sex','age','region'], \
         var_name='year', value_name='value')
 
+    return dataframe
+
+def clean_data(dataframe: pd.DataFrame, country: str) -> pd.DataFrame:
+    """
+    Function responsible for cleaning the data and 
+    filtering it according to a specific region.
+
+    The data cleaning follows this logic: the years are converted 
+    into integers and the values are converted into floats
+    """
+
     # Convert year to int
     dataframe['year'] =  pd.to_numeric(dataframe['year'], errors='coerce').astype(int)
 
```

Comment:
suggestion: perhaps you could have set up a function to take care of all data types

![image](https://user-images.githubusercontent.com/32304147/224570154-fef02948-137d-4962-8791-ca02eec38149.png)

Reply:
Nice suggestion. Makes sense to specify the types for each column, just to make sure everything is correct